plugin/evm/validators_default_test.go

```diff
@@ -66,13 +63,62 @@ func TestGetDefaultValidators_CustomValidators(t *testing.T) {
 	err := os.Setenv("CUSTOM_VALIDATORS", "NodeID-3M9KVT6ixi4gVMisbm5TnPXYXgFN5LHuv,NodeID-NnX4fajAmyvpL9RLfheNdc47FKKDuQW8i")
```

Comment:
You should probably change the name of this test case.
TestNewValidatorsDefault -> TestNewValidatorsDefault_StandardNetwork
TestGetDefaultValidators_CustomValidators -> TestNewValidatorsDefault_CustomNetwork

wdyt?